DeviceSDK/src/main/java/org/astarteplatform/devicesdk/AstartePairingService.java

```diff
@@ -139,7 +139,7 @@ protected List<AstarteTransport> reloadTransports(
           "Null Pointer exception - probably got a wrong payload?", e);
     } catch (Exception e) {
       throw new AstartePairingException(
-          "Failure in calling Pairing API to " + requestUrl.toString(), e);
```

Comment:
Why do you change the `cause` param with string concatenation? I think it is better to propagating the `Throwable`  to the caller.

Reply:
I use this for debug e2e test and forget remove.